.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+
+updates:
+  - package-ecosystem: npm
```

Comment:
Should this not be `cargo`?

Reply:
Good catch. Definitely it must. Somehow didn't save changes before committing. Fixed.